app/controllers/sponsors_controller.rb

```diff
@@ -37,6 +36,14 @@ def create
   end
 
   def update
+    @sponsor = SponsorAcceptedSlackNotifier.new(
```

Comment:
I see what you mean - and I tried doing it - but I decided against it for readability reasons. I want it to be very clear what actually happens when you update a sponsor, while the implementation is hidden away other places.